OpenRA.Mods.RA/Traits/Chronoshiftable.cs

```diff
@@ -41,6 +49,7 @@ public Chronoshiftable(ActorInitializer init, ChronoshiftableInfo info)
 		{
 			this.info = info;
 			self = init.Self;
+			iPositional = self.TraitOrDefault<IPositionable>();
```

Comment:
I assume this would need https://github.com/OpenRA/OpenRA/pull/8848 to work independent from trait ordering.